src/services/map/ol.service.ts

```diff
@@ -87,11 +87,13 @@ export class Openlayers {
     })
   }
 
-  static setLayerOpacity(olMap: OlMap, layerId: LayerId, opacity: number) {
+  setLayerOpacity(olMap: OlMap, layerId: LayerId, opacity: number) {
     const layer = olMap
       .getLayers()
       .getArray()
       .find(layer => layer.get('id') === layerId)
     if (layer) layer.setOpacity(opacity)
   }
 }
+
+export const openLayersService = new OpenLayersService()
```

Comment:
Was there a reason why this was no singleton with static methods before @fgravin ?